### ISSUE TITLE
overwrite results

### ISSUE THREAD

**Issue by sjanssen2** [biocore/wetlab-assistant#8]
The system does not warn when overwriting an existing file. I think we should have a 'force' option that is unset by default for the case that we really want to overwrite something.

**Comment by qiyunzhu** [comment]
@sjanssen2 I found this is how `argparse` does the job. In the very beginning I implemented a homebrew solution that will warn the user if they want to overwrite an existing file. Later I thought: 1) from the user's angle, they may prefer to perform no additional actions (e.g., rename or move the old output file, or specify a new file name). 2) On the other hand, the output file is not going to be placed in archive. The user will immediately copy and paste the entire output file into an Excel spreadsheet, which is in turn the final "mapping file".

**Comment by josenavas** [comment]
@qiyunzhu Just FYI, [click](http://click.pocoo.org/5/) is a really good library for parsing parameters.

**Comment by qiyunzhu** [comment]
@josenavas Thanks! We used click in the [WGS-HGT](https://github.com/biocore/WGS-HGT) project. It is indeed more robust than argparse.

**Comment by wasade** [comment]
argparse is also broken as it is not posix compliant. click actually ships
with a py3 compatible implementation of python2's optparse.

**Comment by mortonjt** [comment]
Hmm. I didn't know that ...

**Comment by wasade** [comment]
I lied, sorry, but reasons are outlined here:

http://click.pocoo.org/5/why/#why-not-argparse

**Comment by mortonjt** [comment]
Yup.  Click is definitely more preferable than argparse.  But @qiyunzhu was under pressure to get something out quickly, and something easy to use.  I mean really easy - no installation required.  This works out of the box without installing _any_ packages.

Like @josenavas mentioned before, we'll deprecate this and move into labadmin.

**Comment by qiyunzhu** [comment]
@wasade Good to know! Thanks!
